docker/mysql.repo

```diff
@@ -0,0 +1,5 @@
+[mysql56-community]
```

Comment:
Do we still need this. What version is in the main repos and how does that tie up with prod. What does prod use?